packages/integration-karma/test/component/spellcheck-attribute/index.spec.js

```diff
@@ -0,0 +1,50 @@
+import { createElement } from 'lwc';
+
+import Container from 'x/container';
+
+function getSpellcheckValue(container, selector) {
+    const elm = container.shadowRoot.querySelector(selector);
+
+    return elm.getAttribute('spellcheck');
+}
+
+describe('spellcheck attribute', () => {
```

Comment:
Nit: This top level describe is unnecessary since we're already printing the directory name when running these tests.